### PR TITLE
SM-7015: Update Registration page

### PR DESCRIPTION
**Pull request by lukeca362**
add charges article with table of charges and update registration page

### CODE REVIEW

File: docs/registration/index.md [hunk 1]
```diff
@@ -5,51 +5,66 @@ title: Registration
...
+    if your organisation is a Highway Authority or works Promoter, you must have a <a class="govuk-link" href="https://services.geoplace.co.uk/street/swa-codes">Street Works Act (SWA) code</a> confirming your legal right to carry out works. Alternatively you can <a class="govuk-link" href="https://services.geoplace.co.uk/street/registration">create a  SWA code</a> via Geoplace.
+  </li>
 </ol>
 
-## When can you sign up?
+## What sort of user am I?
 {: .govuk-heading-m}
 
-From 1st November, Street Manager enters into Public Beta phase, when the service is open to everyone to use. This means that you can access the Sandbox test environment as well as the Production environment which is where real data will live.  Everyone who registers to sign up and completes the registration process will be added to a waitlist. Access will then be granted to each organisation from in order of registration completion.
+Street Manager has 4 types of user for which access may or may not be a charged. The user types are:
```
Comment (ccurran01) [inline docs/registration/index.md:36]:
```suggestion
Street Manager has 4 types of user for which access may or may not be charged. The user types are:
```

File: docs/_articles/charges.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+{: .govuk-heading-xl}
+## Street Manager Charges
+{: .govuk-heading-m}
+
+How are Street Manager funds used?
+{: .govuk-heading-s}
+
+The DfT has invested over £10m in the development of Street Manager. The service is now funded through user charges. These charges are used to fund the costs involved in maintaining the service and for its continuous improvement.<br>
+The charges are fixed for the first 2 years of operation. In mid-2021, we will have a full year of Street Manager data and can discuss both changes to the methodology of calculating the charges and if we want to reduce or increase the speed of development of Street Manager functionality.<br>
+To be clear, charges for Street Manager will be used to fund the maintenance and continuous improvement of the service. They are on a cost recovery basis. Any surplus that may arise will be used to improve Street Manager.<br>
```
Comment (ccurran01) [inline docs/_articles/charges.md:15]: Not a big deal - I think `{: .govuk-body}` would have the same effect here? 
```suggestion
To be clear, charges for Street Manager will be used to fund the maintenance and continuous improvement of the service. They are on a cost recovery basis. Any surplus that may arise will be used to improve Street Manager.
{: .govuk-body}
```

File: docs/_articles/charges.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+Charges are  by organisation (local highway authority and statutory undertaker). There are no restrictions on the number of users, once your organisation has signed up. Any future development and new services will be made available to all users at no additional cost, and each organisation will be charged whether you use Street Manager via the user interface or via an API.
```
Comment (ccurran01) [inline docs/_articles/charges.md:16]:
```suggestion
Charges are by organisation (local highway authority and statutory undertaker). There are no restrictions on the number of users, once your organisation has signed up. Any future development and new services will be made available to all users at no additional cost, and each organisation will be charged whether you use Street Manager via the user interface or via an API.
```

File: docs/registration/index.md (hunk 1 above)
Comment (lukeca362) [reply docs/registration/index.md:36]: Nice one! 👌🏻

File: docs/_articles/charges.md (hunk 3 above)
Comment (lukeca362) [reply docs/_articles/charges.md:16]: 👌🏻

File: docs/_articles/charges.md (hunk 2 above)
Comment (lukeca362) [reply docs/_articles/charges.md:15]: Yeah, I was trying to break up the block of text and the `{: .govuk-body}` adds some spacing. Thanks

**Review by ccurran01 — APPROVED** [review]

**Review by RichardEustace — APPROVED** [review]

**Pull request merged**